### PR TITLE
test: options for tiers in NewSocialEvent helper func

### PR DESCRIPTION
**Pull request by xcoulon**
changes to be used in https://github.com/codeready-toolchain/host-operator/pull/706

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

**Review by alexeykazakov — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]

File: pkg/test/socialevent/socialevent.go [hunk 1]
```diff
@@ -47,3 +47,15 @@ func WithActivationCount(value int) Option {
 		event.Status.ActivationCount = value
 	}
 }
+
+func WithUserTier(tier string) Option {
```
Comment (rajivnathan) [inline pkg/test/socialevent/socialevent.go:51]: Just noticed there's no `WithMaxAttendees` modifier. Are we missing some testing in that area?

Comment (xcoulon) [reply pkg/test/socialevent/socialevent.go:51]: good point, let me check that

Comment (xcoulon) [reply pkg/test/socialevent/socialevent.go:51]: added in 302b374

**Comment by xcoulon** [comment]
@rajivnathan I've taken the liberty of merging the PR after adding https://github.com/codeready-toolchain/toolchain-common/commit/302b374718174a7244b69fa5832bf40c2a8feff6 so I can move on with https://github.com/codeready-toolchain/host-operator/pull/706

**Pull request merged**